x/ecocredit/simulation/operations.go

```diff
@@ -0,0 +1,594 @@
+package simulation
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/simapp/helpers"
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+	"github.com/regen-network/regen-ledger/types/math"
+	"github.com/regen-network/regen-ledger/x/ecocredit"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	regentypes "github.com/regen-network/regen-ledger/types"
+)
+
+// Simulation operation weights constants
+const (
+	OpWeightMsgCreateClass = "op_weight_msg_create_class"
+	OpWeightMsgCreateBatch = "op_weight_msg_create_batch"
+	OpWeightMsgSend        = "op_weight_msg_send"
+	OpWeightMsgRetire      = "op_weight_msg_retire"
+	OpWeightMsgCancel      = "op_weight_msg_cancel"
+)
+
+// ecocredit operations weights
+const (
+	WeightCreateClass = 100
+	WeightCreateBatch = 80
```

Comment:
I think we want to generate a lot more classes than batches, so maybe this weight should be higher?

Reply:
Yes, we are generating more classes than batches

```
"ecocredit": {
  "/regen.ecocredit.v1alpha1.MsgCreateBatch": {
   "ok": 73,
   "failure": 0
  },
  "/regen.ecocredit.v1alpha1.MsgCreateClass": {
   "ok": 118,
   "failure": 0
  }
 }
 ```